drivers/linux/amd64/cpu.asm

```diff
@@ -419,7 +419,11 @@ __swsmi__:
     mov rdi, r12 ; rdi_value
 
     ; this OUT instruction will write WORD value (smi_code_data) to ports 0xB2 and 0xB3 (SW SMI control and data ports)
-    out 0B2h, ax
+    push rax

```

Comment:
nit: isn't is the same  to do
`out 0b3h, ah`
`out 0b2h, al`

Reply:
It's more of that there's no instruction encoding for OUT using AH. Only AL, AX and EAX can be used.